--- libjpeg-turbo-3.0.3/CMakeLists.txt.orig	2024-05-08 19:25:20.015726194 +0200
+++ libjpeg-turbo-3.0.3/CMakeLists.txt	2024-05-08 19:27:05.309401756 +0200
@@ -229,14 +229,10 @@
   "Include arithmetic encoding support when emulating the libjpeg v6b API/ABI"
   TRUE)
 boolean_number(WITH_ARITH_ENC)
-if(CMAKE_C_COMPILER_ABI MATCHES "ELF X32")
-  set(WITH_JAVA 0)
-else()
   option(WITH_JAVA
     "Build Java wrapper for the TurboJPEG API library (implies ENABLE_SHARED=1)"
     FALSE)
   boolean_number(WITH_JAVA)
-endif()
 option(WITH_JPEG7
   "Emulate libjpeg v7 API/ABI (this makes ${CMAKE_PROJECT_NAME} backward-incompatible with libjpeg v6b)"
   FALSE)
